app/models/user.rb

```diff
@@ -7,4 +7,10 @@ class User < ActiveRecord::Base
                     uniqueness: { case_sensitive: false }
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
+
+  def User.digest(string)
+    cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
```

Comment:
__[rubocop]__
* Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead. [:ref](https://github.com/bbatsov/ruby-style-guide#no-multiline-ternary)